lesson_03/script.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+const ul = document.getElementById("js-target");
+for (let i = 1; i <= 2; i += 1) {
```

Comment:
IMO
What is 2 of 2? It's a [magic number](https://ja.wikipedia.org/wiki/%E3%83%9E%E3%82%B8%E3%83%83%E3%82%AF%E3%83%8A%E3%83%B3%E3%83%90%E3%83%BC_(%E3%83%97%E3%83%AD%E3%82%B0%E3%83%A9%E3%83%A0)), so let's refrain from it.